app/i18n/locales/en.json

```diff
@@ -568,14 +570,14 @@
   "Upload_file_question_mark": "Upload file?",
   "User": "User",
   "Users": "Users",
-  "User_added_by": "User {{userAdded}} added by {{userBy}}",
+  "User_added_by": "added {{userAdded}}",
   "User_Info": "User Info",
   "User_has_been_key": "User has been {{key}}",
   "User_is_no_longer_role_by_": "{{user}} is no longer {{role}} by {{userBy}}",
-  "User_muted_by": "User {{userMuted}} muted by {{userBy}}",
-  "User_removed_by": "User {{userRemoved}} removed by {{userBy}}",
+  "User_muted_by": "muted {{userMuted}}",
+  "User_removed_by": "removed {{userRemoved}}",
   "User_sent_an_attachment": "{{user}} sent an attachment",
-  "User_unmuted_by": "User {{userUnmuted}} unmuted by {{userBy}}",
+  "User_unmuted_by": "unmuted {{userUnmuted}}",
```

Comment:
Hey man, can you review these parameters? Because they are different between PT-BR and En.